CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [4.41.3](https://github.com/voxpupuli/beaker/tree/4.41.3) (2024-03-27)
+
+[Full Changelog](https://github.com/voxpupuli/beaker/compare/4.41.2...4.41.3)
+
+**Implemented enhancements:**
+
+- \[PE-37978\] Add 'amazon' to repo_filename method with tests [\#1861](https://github.com/voxpupuli/beaker/pull/1861) ([span786](https://github.com/span786))
```

Comment:
There's another enhancement that should be listed here: https://github.com/voxpupuli/beaker/compare/4.41.2...4.x

Reply:
Updated the changelog with all the enhancements